Jotify/SceneDelegate.swift

```diff
@@ -5,18 +5,37 @@
 //  Created by Harrison Leath on 1/16/21.
 //
 
-import FirebaseDynamicLinks
 import SwiftUI
 import UIKit
+import Airbridge
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate, UNUserNotificationCenterDelegate {
     var window: UIWindow?
+    
+    private func handleDeepLink(_ url: URL) {
+        let urlString = url.absoluteString
+        
+        if urlString.hasPrefix("jotify://recentnotewidget-link") {
+            handleRecentNoteWidget()
+        } else if urlString.hasPrefix("jotify://settings") {
+            handleSettings(urlString)
+        } else if urlString.hasPrefix("jotify://write") {
+            presentController(WriteNoteController())
+        } else if urlString.hasPrefix("jotify://note") {
+            handleNoteEdit(urlString)
+        } else {
+            guard let components = URLComponents(url: url, resolvingAgainstBaseURL: false),
+                  let queryItems = components.queryItems else { return }
+            
+            for queryItem in queryItems {
+                if queryItem.name == "referralId" {
+                    UserDefaults.standard.set(queryItem.value, forKey: "referralId")
+                }
+            }
+        }
+    }
 
```

Comment:
The handleDeepLink method has been refactored to handle both Airbridge and existing deep links. This unified approach ensures consistent deep link handling while maintaining backward compatibility with existing features like widget links and settings navigation.